tests/Integration/Infrastructure/Controller/Regulation/PublishRegulationControllerTest.php

```diff
@@ -15,7 +15,7 @@ final class PublishRegulationControllerTest extends AbstractWebTestCase
 
     public function testPublish(): void
     {
-        $client = $this->login();
+        $client = $this->login('mathieu.fernandez@beta.gouv.fr');
```

Comment:
Utiliser les constantes qui indiquent de quoi on parle

```suggestion
        $client = $this->login(UserFixture::MAIN_ORG_ADMIN_EMAIL);
```